### PR TITLE
Added shell script to automate test execution pre-setup steps

### PR DESCRIPTION
**Pull request by nitishSr**
This script is for combining the pre-setup steps that are needed for executing any test case. These steps include :

1. Starting the MTA web console
2. Setup and activate python 3 virtual environment
3. Start selenium container
4. Start the vnc viewer
5. Close the selenium container and vnc after test execution

The script accepts an arg named t which corresponds to the full path of test case from main directory.
Things to consider for more up-gradation -

To add ftp setup
Handling of env local yaml file automatically based on some extra inputs

### CODE REVIEW

File: exec_test.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+then
+    echo -e "\n MTA web console container is already running !"
+else
+    if [[ $(docker ps -a | grep windup_mta) = *windup_mta* ]]
+    then
+        echo -e "\n Starting the existing MTA web console docker container !"
+        docker start windup_mta
+    else
+        echo -e "\n Starting new MTA web console container !"
+        docker run -d -p 8080:8080 --name windup_mta -it mta
```
Comment (sshveta) [inline exec_test.sh:42]: I am getting error here that the image does not exist .
While building we need to specify image name I think

**Review by sshveta — CHANGES_REQUESTED** [review]
Looks Good Nitish . It is a helpful set up script. 
I am feeling that it has become little more complicated though ..
It's mainly to be used by team members for setting up automation and also may be in jenkins job and then user can execute any test.

1) We can rename it to something more intuitive like "setup_mta.sh"
2) We can give some prerequisites like *mta not running and * selenium not running before using this script .
3) we can give it a default test case to execute so -t parameter won't be needed.

File: exec_test.sh [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+echo -e "\n MTA web console is available now ! \n"
+
+#Setup python3 virtual environment (version issue needs to be sorted)
+python3.7 -m venv .mta_venv
+source ./.mta_venv/bin/activate
+pip install -e .
+
+#Start selenium container
+if [[ $(docker ps | grep mta_selenium) = *mta_selenium* ]]
```
Comment (sshveta) [inline exec_test.sh:69]: I don't think we need if else here.
If we do "mta selenium start" and the server is already running it just gives message "server already running "

**Comment by nitishSr** [comment]
Understood the idea of setup, will make changes as suggested, and instead of running any test case from here, I will just restrict it to setup script for now. We can decide on it later and then discuss on running the test.

**Review by sshveta — COMMENTED** [review]
Looks good Nitish :+1: 
Let's discuss about ftp thing too

File: setup_mta.sh [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    echo -e "\n mta selenium container is already running !"
+else
+    echo -e "\n Starting mta selenium container !"
+    mta selenium start
+    sleep 10
+fi
+
+#Start tiger vnc viewer
+mta selenium viewer
+
```
Comment (ganeshhubale) [inline setup_mta.sh:78]: we can use `mta selenium start -w`

Comment (nitishSr) [reply setup_mta.sh]: Its working, removed the lines and replaced with this single command

**Review by sshveta — APPROVED** [review]

**Pull request merged**